kgraphql/src/main/kotlin/com/apurebase/kgraphql/schema/execution/ArgumentTransformer.kt

```diff
@@ -42,6 +42,10 @@ open class ArgumentTransformer(val schema : DefaultSchema) {
                     params.getValue(valueField.name.value) to transformValue(paramType, valueField.value, variables)
                 }.toMap()
 
+                require(params.size == valueMap.size) {
```

Comment:
This only validates that the number of parameters match. We don't want this.

We want to make sure that all params that have `isOptional` as `false` need to be provided within the `valueMap`